lib/round.rb

```diff
@@ -0,0 +1,42 @@
+class Round
+  attr_reader :deck, :turns, :number_correct
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+    @right_answers = []
+  end
+
+  def current_card
+    @curr_card = @deck.cards[turns.count]
+  end
+
+  def take_turn(guess)
+    @new_turn = Turn.new(guess, current_card)
+    @turns << @new_turn
+
+    if @new_turn.correct?
+      @number_correct += 1
+      @right_answers << @curr_card
+    end
+
+    @new_turn
+  end
+
+  def number_correct
+    @right_answers.count
+  end
```

Comment:
I feel like `number_correct`, `@number_correct`, and `@right_answers` are all doing a little bit of the same thing with some very minor variances. I'd maybe try to refactor to see if you can consolidate that a little bit.